arc/nvwcpp.c

```diff
@@ -125,10 +120,10 @@ int main(int argc, char** argv) {
 #endif
 
 	int br = totalLength;
-	double* sol = new (std::nothrow) double[br];
-	double* rad = new (std::nothrow) double[br];
+	double* sol = (double*) malloc(br*sizeof(double));
+	double* rad = (double*) malloc(br*sizeof(double));
 
-	if (!sol or !rad){
```

Comment:
`or` is a C++ digraph which has been [scheduled for *removal* in C++17](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n3981.html)!